src/queries/role.js

```diff
@@ -26,16 +26,28 @@ function queryAllByRole(
 
 const getMultipleError = (c, role) =>
   `Found multiple elements with the role "${role}"`
-const getMissingError = (container, role) =>
-  `
-Unable to find an element with the role "${role}"
 
+const getMissingError = (container, role) => {
+  const roles = prettyRoles(container)
+  let roleMessage
+
+  if (roles.length === 0) {
+    roleMessage = 'There are no available roles.'
+  } else {
+    roleMessage = `
 Here are the available roles:
 
   ${prettyRoles(container)
```

Comment:
We can avoid calculating the `prettyRoles` twice by putting `roles` in here instead of calling `prettyRoles(container)` again.

Reply:
Oops! I meant to do that. Haha.

Reply:
Corrected!